components/match2/commons/match_group_input.lua

```diff
@@ -583,7 +654,161 @@ function MatchGroupInput.readMvp(match)
 	return {players = parsedPlayers, points = mvppoints}
 end
 
+---@param match table
+---@param opponentIndex integer
+---@param opponentInput table
+---@return {players: table[], substitutions: MatchGroupInputSubstituteInformation[]}
+function MatchGroupInput.extractManualPlayersInput(match, opponentIndex, opponentInput)
+	--players from manual input as `opponnetX_pY`
+	local players = Array.mapIndexes(function(playerIndex)
+		return Json.parseIfString(match['opponent' .. opponentIndex .. '_p' .. playerIndex])
+	end)
+	local manualInput = {players = players}
```

Comment:
We can drop this I think, I'm only aware of 2 pages using it and I asked @iMarbot to replace it the sub style instead

Reply:
Also current implemenation here does match usage, as they can be `opponent1_p3` as the only input, which would fail here

Reply:
can't find any usage at all, so will kick